packages/seacas/libraries/ioss/src/exodus/Ioex_ChangeSet.h

```diff
@@ -0,0 +1,69 @@
+// Copyright(C) 1999-2020, 2022, 2023, 2024 National Technology & Engineering Solutions
+// of Sandia, LLC (NTESS).  Under the terms of Contract DE-NA0003525 with
+// NTESS, the U.S. Government retains certain rights in this software.
+//
+// See packages/seacas/LICENSE for details
+
+#pragma once
+
+#include "Ioss_CodeTypes.h"
+#include "Ioss_DBUsage.h"    // for DatabaseUsage
+#include "Ioss_ChangeSet.h"  // for ChangeSet
+#include "Ioss_ChangeSetFactory.h"  // for ChangeSetFactory
+#include "Ioss_Map.h"        // for Map
+#include <cstddef>           // for size_t
+#include <cstdint>           // for int64_t
```

Comment:
Might not need this.

Reply:
ChangeSets is an interface that is dependent on the implementation of the internal file format. Plain IOSS ChangeSet only handles the linear and cyclic versions. The Exodus ChangeSet handles the `group` implementation and other file formats such as catalyst which may implement their own versions.